kwasm-lemmas.md

```diff
@@ -325,8 +347,15 @@ They are non-trivial in their implementation, but the following should obviously
       requires WIDTH ==Int 1
       [simplification]
 
-    rule #getRange(BM, ADDR, WIDTH) modInt 256 => #get(BM, ADDR)
-      requires notBool (WIDTH ==Int 0)
+    rule #getRange(BM, ADDR, WIDTH) modInt BYTE_SIZE => #get(BM, ADDR)
+      requires BYTE_SIZE ==Int 256
+       andBool notBool (WIDTH ==Int 0)
```

Comment:
This should probably be `notBool (WIDTH <=Int 0)`

Reply:
Fixed